<launch>

    <node pkg="sensors" type="ins_2d" name="ins_2d" />

    <node pkg="sensors" type="pid_fl" name="pid_fl" />

    <node pkg="sensors" type="ardumotors.py" name="ardumotors" />

    <node pkg="sensors" type="cs_ch_2.py" name="cs_ch_2" />

    <node pkg="rosserial_python" type="serial_node.py" name="serial_node" args = "/dev/ttyUSB1" />

    <!--Launch rosbag to record topics -->
    <node type="record" pkg="rosbag" name="recorder" args="record -a -o /home/ubuntu/catkin_ws/src/sensors/bags/pid_fl/u_1_psi_ref" />

</launch>
